### DISCUSSION TITLE
Presales?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#1533] [General]
Is there any presale scheduled to happen before officially get in coingecko etc? What is the schedule/roadmap ahead?

**Comment by hoffmang9** [comment]
The Zooms and Business Whitepaper answer these questions https://chia.net/2021/02/10/chia-businesss-whitepaper.html

**Reply by GeorgePapageorgakis** [reply]
Nice but I do not see any scheduled dates for listing etc. Will you announce the listing in coingencko or uniswap/binance etc  days before the listing or after the listing has been fulfilled?

**Reply by hoffmang9** [reply]
As we said in the Whitepaper and Botha Zooms that I linked you to, we can't discuss dates.
